@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -1434,6 +1453,8 @@ describe("MapView", function() {
             };
         }
         mapView = new MapView({ canvas, theme: {} });
+        await mapView.getTheme();
+        await mapView.sceneEnvironment.backgroundDataSourcePromise;
```

Comment:
Not needed, I checked locally and only waiting for the theme it's enough to get rid of the errors.

Reply:
I will recheck, because I tried this too and got another issue.

Reply:
I was wrong, have fixed.